experiment/depth-first-traversal/simulation/js/treePractice.js

```diff
@@ -57,7 +57,16 @@ function check(d){
     if(tree_traversal.sequence_list[index]){
   
     document.getElementById("commentbox").style.display = "block";
+
+    if(document.getElementById("trav").value==="inorder"){
```

Comment:
Since document.getElementById("trav").value and document.getElementById("comments") are repeated multiple times, it'd be better to take them out in variables. It'd unclutter the code and increase readability. Something like:

const traversalType = document.getElementById("trav").value;

let hintText = 'Wrong sequence.Try again! \n HINT: Next node in traversal: '+tree_traversal.sequence_list[index];

if (traversalType === "inorder") {
hintText = 'Wrong sequence.Try again! \n HINT:The traversal is : Inorder(Left,Root,Right)';
} else if (traversalType === "preorder") {
hintText = 'Wrong sequence.Try again! \n HINT:The traversal is : Preorder(Root,Left,Right)';
}...


document.getElementById("comments").innerHTML = hintText;

Reply:
I have updated the code

Reply:
Please let me know if you have any issues